iothub_client/src/iothubtransport_amqp_twin_messenger.c

```diff
@@ -2005,6 +2005,7 @@ int twin_messenger_stop(TWIN_MESSENGER_HANDLE twin_msgr_handle)
         }
         else
         {
+            twin_msgr->subscription_error_count = 0;
```

Comment:
interesting this wasn't here before. Why is it only now that it's needed? Seems more universally true.

Reply:
The lack of that line was apparently a bug.

Reply:
There was a github issue complaining about that.